cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -138,7 +138,6 @@ func TestAddFlags(t *testing.T) {
 				EnableProfiling:                                 false,
 				CIDRAllocatorType:                               "CloudAllocator",
 				NodeCIDRMaskSize:                                48,
-				ServiceSyncPeriod:                               metav1.Duration{Duration: 2 * time.Minute},
```

Comment:
Shouldn't we clean up line 103 as well? ("--service-sync-period=2m",)

Reply:
Yes. I am working on flushing out out of tests.